Source/Controllers/SubviewControllers/SPProcessListController.m

```diff
@@ -244,15 +247,16 @@ - (IBAction)saveServerProcesses:(id)sender
                 
                 for (NSDictionary *process in self->processesFiltered)
                 {
-                    NSString *stringTmp = [NSString stringWithFormat:@"%@ %@ %@ %@ %@ %@ %@ %@",
+                    NSString *stringTmp = [NSString stringWithFormat:@"%@ %@ %@ %@ %@ %@ %@ %@ %@",
                                            [process objectForKey:@"Id"],
                                            [process objectForKey:@"User"],
                                            [process objectForKey:@"Host"],
                                            [process objectForKey:@"db"],
                                            [process objectForKey:@"Command"],
                                            [process objectForKey:@"Time"],
                                            [process objectForKey:@"State"],
-                                           [process objectForKey:@"Info"]];
+                                           [process objectForKey:@"Info"],
+                                           [process objectForKey:@"Progress"]];
```

Comment:
I noticed this isn't a safeObjectFroKey accessing here or in the other place in this controller - can we confirm it won't crash for MySQL responses?

Reply:
I already tested on MariaDB/MySql versions and have no crash issues.

Reply:
I must have missed that, my bad! Nice work on this one!